src/platform/intel/cavs/platform.c

```diff
@@ -155,14 +155,9 @@ const struct ext_man_windows xsram_window
 #endif
 
 #if CONFIG_TIGERLAKE
-#if CONFIG_CAVS_LPRO_ONLY
-#define CAVS_DEFAULT_RO		SHIM_CLKCTL_RLROSCC
-#define CAVS_DEFAULT_RO_FOR_MEM	SHIM_CLKCTL_OCS_LP_RING
-#else
```

Comment:
I deleted CONFIG_CAVS_LPRO_**ONLY** which is for debugging only at the moment, choose n if unclear

Reply:
yeah, that was just a renaming in https://github.com/thesofproject/sof/pull/3205/commits/59c3315a4f6fb329104540a6c957bc2901ff0e91 so from those commits it looks like this was a debug option also available for TGL. It's ok if we remove it because we don't need or don't want it any more, but at least I don't see that it was only available for CFL

Reply:
@lyakh   I did involved in this feature and you may recall it. In 2019, we found a HW hang on HDA platform only on CFL and then confirmed by HW team that we need to use LPO for HW restraint. But we don't need it for CML and later HW platform.

Reply:
Ok, good, maybe it would just be a bit clearer if we said something like "remove this debug option because we don't need it any more" in the commit message - just by looking at the patch nothing in it explains why it's only relevant up to CFL